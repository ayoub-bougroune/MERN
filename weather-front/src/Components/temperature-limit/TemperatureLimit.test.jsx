import React from 'react';
import renderer from 'react-test-renderer';
import TemperatureLimit from './index';

test('The Carousel renders correctly', () => {
  const limit = renderer
    .create(<TemperatureLimit />);
  const tree = limit.toJSON();
  expect(tree).toMatchSnapshot();
});