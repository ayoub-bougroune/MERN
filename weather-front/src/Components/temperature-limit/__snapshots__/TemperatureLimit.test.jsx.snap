// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Carousel renders correctly 1`] = `
<div
  className="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column"
>
  <div
    className="MuiGrid-root"
  >
    <div
      className="makeStyles-arrowUp-1"
    />
  </div>
  <div
    className="MuiGrid-root"
  />
  <div
    className="MuiGrid-root"
    color="#f00"
  >
    <div
      className="MuiBox-root MuiBox-root-107"
    />
  </div>
</div>
`;
